src/game.cpp

```diff
@@ -4558,10 +4577,42 @@ void Game::showPauseMenu()
 	os		<< "button_exit[4," << (ypos++) << ";3,0.5;btn_exit_os;"
 		<< strgettext("Exit to OS")   << "]"
 		<< "textarea[7.5,0.25;3.9,6.25;;" << control_text << ";]"
-		<< "textarea[0.4,0.25;3.5,6;;" << PROJECT_NAME_C "\n"
-		<< g_build_info << "\n"
-		<< "path_user = " << wrap_rows(porting::path_user, 20)
-		<< "\n;]";
+		<< "textarea[0.4,0.25;3.9,6.25;;" << PROJECT_NAME_C " " VERSION_STRING "\n"
+		<< "\n"
+		<<  strgettext("Game info:") << "\n";
+	const std::string &address = client->getAddressName();
+	static const std::string mode = strgettext("- Mode: ");
+	if (!simple_singleplayer_mode) {
+		Address serverAddress = client->getServerAddress();
+		if (address != "") {
+			os << mode << strgettext("Remote server") << "\n"
+					<< strgettext("- Address: ") << address;
+		} else {
+			os << mode << strgettext("Hosting server");
+		}
+		os << "\n" << strgettext("- Port: ") << serverAddress.getPort() << "\n";
+	} else {
+		os << mode << strgettext("Singleplayer") << "\n";
+	}
+	if (simple_singleplayer_mode || address == "") {
+		static const std::string on = strgettext("On");
+		static const std::string off = strgettext("Off");
+		std::string damage = g_settings->getBool("enable_damage") ? on : off;
```

Comment:
is this possible to use const ref for the 5 std::string there or compiler doesn't like it ?